dataeng/jobs/analytics/TableauConfigBackup.groovy

```diff
@@ -0,0 +1,52 @@
+package analytics
+import static org.edx.jenkins.dsl.AnalyticsConstants.secure_scm
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_log_rotator
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_wrappers
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_publishers
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_triggers
+import static org.edx.jenkins.dsl.AnalyticsConstants.secure_scm_parameters
+
+class TableauConfigBackup{
+    public static def job = { dslFactory, allVars ->
+        dslFactory.job("tableau-config-backup"){
+            logRotator common_log_rotator(allVars)
+            parameters secure_scm_parameters(allVars)
+            multiscm secure_scm(allVars)
+            parameters {
+                stringParam('ANALYTICS_TOOLS_URL', allVars.get('ANALYTICS_TOOLS_URL'), 'URL for the analytics tools repo.')
+                stringParam('ANALYTICS_TOOLS_BRANCH', allVars.get('ANALYTICS_TOOLS_BRANCH'), , 'Branch of analytics tools repo to use.')
+                stringParam('TABLEAU_SERVER_IP', allVars.get('TABLEAU_SERVER_IP'), 'IP address of Tableau Server.')
```

Comment:
I can't recall, did you say there was a reason why we can't use a hostname? Unless we can't I think it's preferable.

Reply:
Yes in case of jenkins and dbt-docs server, we can ssh into server using their DNS name. But in case of Tableau [tableau.analytics.edx.org](https://tableau.analytics.edx.org/) its Route53 DNS record points to public load balancer. This load balancer only allow traffic port on 80, 8080, 8850, 8060 which are the access points of Tableau and Tableau Server Manager. I am also in favor of not exposing port 22(ssh port) on public load balancer. @estute  

![image](https://user-images.githubusercontent.com/30023768/164311872-f9d7c5f8-4f3f-43d7-9330-6dbd63351a01.png)

Reply:
Ideally the load balancer itself would also use a DNS record to resolve the IP address of the backend service (tableau).  Is that also not the case?